config/core/build-configs.yaml

```diff
@@ -939,6 +986,26 @@ build_configs:
     branch: 'v5.15-rt'
     variants: *preempt_rt_variants
 
+  rust:
+    tree: mainline
+    branch: 'master'
+    variants:
+      rustc-1.62:
+        build_environment: rustc-1.62
+        fragments: [rust, rust-samples]
+        architectures:
+          x86_64: *x86_64_arch
+
+  rust-for-linux:
+    tree: rust-for-linux
```

Comment:
Is it possible to add rust-next too (i.e. is the one that is in linux-next)? If it is a matter of resources, it is probably better to replace this one (`rust`) with it.

Reply:
I think it's  possible, however I'd rather have it added with another PR.
@gctucker What do you recon?